tests/python/unit/test_binary_model.py

```diff
@@ -0,0 +1,44 @@
+import sys
+sys.path.append("..")
+print('\n'.join(sys.path))
+from generalized_linear_models.dku_glm import BinaryClassificationGLM
```

Comment:
Nitpick: would be cleaner to order imports https://www.python.org/dev/peps/pep-0008/#:~:text=Imports%20are%20always%20put%20at,Related%20third%20party%20imports.